components/zcash_note_encryption/README.md

```diff
@@ -12,6 +12,15 @@ users with their own existing types can similarly implement the trait themselves
 [`zcash_primitives`]: https://crates.io/crates/zcash_primitives
 [`orchard`]: https://crates.io/crates/orchard
 
+## Cross-Workspace Dependency Cycle
+
+There is a cross-workspace cyclic dependency: `zcash_client_backend` (local)
+depends on [`orchard`](https://github.com/zcash/orchard) which depends on this crate
+`zcash_note_encryption`. For this reason `zcash_client_backend` does not depend on the local
+`zcash_note_encryption` crate, but rather the published crate release.
```

Comment:
That isn't cyclic, right? The issue is that `zcash_client_backend` might depend on a different version of `zcash_note_encryption` to `orchard`; not that there is any cycle in the dependency graph (which Rust and `cargo` do not support).
```suggestion
There is a cross-workspace dependency that requires care with versioning:
`zcash_client_backend` (local) depends on [`orchard`](https://github.com/zcash/orchard)
which depends on this crate `zcash_note_encryption`. For this reason `zcash_client_backend`
does not depend on the local `zcash_note_encryption` crate, but rather the published crate
release.
```